net/BigBuffers.Tests/BigBuffers.Tests.csproj

```
@@ -28,5 +38,8 @@
     <ProjectReference Include="..\BigBuffers.Runtime\BigBuffers.Runtime.csproj" />
     <ProjectReference Include="..\BigBuffers.Tests.Monster\BigBuffers.Tests.Monster.csproj" />
   </ItemGroup>
-
+  <Target Name="GenerateHtmlCoverageReport" AfterTargets="GenerateCoverageResultAfterTest">
+    <!-- generate markdown coverage report and badges -->
+    <ReportGenerator ReportFiles="@(CoverletReport)" ReportTypes="MarkdownSummary;Badges" SourceDirectories="../BigBuffers.Runtime" TargetDirectory="../../coverage/" />
```

```suggestion
    <ReportGenerator ReportFiles="@(CoverletReport)" ReportTypes="MarkdownSummary;Badges" SourceDirectories="../BigBuffers.Runtime" TargetDirectory="../coverage/" />
```
Just up to the solution directory instead of to the repository root.